tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -693,9 +754,27 @@ private static class CommandFileReport {
     private static class CommandRunWorkflows {
         @Parameter(names = "--help", description = "Prints help for run-workflows-through-wes", help = true)
         private boolean help = false;
-        @Parameter(names = "--config-file-path", description = "Path to config file (default ./tooltesterConfig.yml")
+        @Parameter(names = "--config-file-path", description = "Path to config file")
         private String configFilePath = "tooltesterConfig.yml";
 
+        @Parameter(names = "--directory-to-save-results-to", description = "Name of the directory you want to save the run results to")
```

Comment:
This is nitpicky and personal preference, and we'll probably take the defaults anyway when running it, but I'd find it easier it you'd use the same command-line switch for both writing and reading. For example, `--results-dir` here and on line 775.

Less to remember...

Reply:
Fixed [26b07fb](https://github.com/dockstore/dockstore-support/pull/461/commits/26b07fb4b3390274a176cde167713dd82e5355c7)